### ISSUE TITLE
Estimate on sprint includes tasks

### ISSUE THREAD

**Issue by richardzaat** [cschleiden/azure-boards-estimate#53]
When estimating a complete sprint, the tasks are also included for estimation

**Comment by Tiberriver256** [comment]
Yes, definitely makes it tougher to estimate. If other teams need or want this could you add it to the settings to allow different work items to be included in the estimation?

**Comment by loehard** [comment]
Hi @cschleiden 
A little sugestion below: 
Would be possible to implement something like this? 
The tasks of each PBI would have the option to be collapsed
![image](https://user-images.githubusercontent.com/11324159/61069875-ab65f400-a3e3-11e9-9ca4-66d50437285b.png)

**Comment by jvduval** [comment]
I can enter estimates for the PBI, but I would like to estimate "each" individual task and have that ROLLUP to the PBI.  The issue, is I get an error trying to estimate a "task", the error says "Cannot save estimate, no field is configured for work items of type Task", how can I fix/implement this method using your estimation tool?

**Comment by fabcap83** [comment]
I solved this issue by customizing the Task configuration and adding the existing field Size. 
I also added a new rule on tasks which copies the value of Size to the Original Estimate and Remaining Work as this would considered for the in Azure DevOps built-in capacity planning. This might depend on us using a CMMI process.
